tests/unit/type/hash.tcl

```diff
@@ -483,6 +483,7 @@ start_server {tags {"hash"}} {
     } {1 0 1 0}
 
     test {Is a ziplist encoded Hash promoted on big payload?} {
+        r config set hash-max-listpack-value 64
```

Comment:
OK. We have many like this already, so I can accept it.

The proper solution would be that every test restores the config after they tested something with a different config. They should do CONFIG GET to get the old value and use it to restore the config afterwards. But we don't have that and it makes no sense to have it in a few tests if we don't have it in all tests.